test_station.py

```diff
@@ -47,4 +47,15 @@ def test_inconsistent_typical_range_stations():
     town = "My Town"
     wrs = MonitoringStation(s_id, m_id, label, coord, trange, river, town)
 
-    assert (inconsistent_typical_range_stations([ntrs, wrs]) == [ntrs, wrs])
+    # Create a station
+    s_id = "test-s-id"
+    m_id = "test-m-id"
+    label = "some station"
+    coord = (-2.0, 4.0)
+    trange = (-2.3, 3.4445)
+    river = "River X"
+    town = "My Town"
+    s = MonitoringStation(s_id, m_id, label, coord, trange, river, town)
+
+    assert (inconsistent_typical_range_stations([ntrs, wrs, s]) == [ntrs, wrs, s])
```

Comment:
`s` should not be included in the output because `s` has a typical range

Reply:
ok